spec/mongoid/persistable/settable_spec.rb

```diff
@@ -315,6 +315,25 @@
         end
       end
 
+      context 'when a leaf value in the nested hash is updated to a number' do
+
+        let(:church) do
+          Church.new.tap do |a|
+            a.location = {'address' => {'city' => 'Berlin', 'street' => 'Yorckstr'}}
+            a.name = 'Church1'
+            a.save
+          end
+        end
+
+        before do
+          church.set('location.address.city' => 12345)
```

Comment:
If I'm understanding correctly, this is where the exception would have occurred before the change? If so, let's put this inside a `let` block instead and invoke inside the `it` block below so that any future regression occurs in a place that's more easily identified.

Reply:
Can do, but this way it's more consistent with all the examples around it. I think an exception raised anywhere in the spec is fine for failing a test; it doesn't necessarily need to be raised in the `it` block.

Reply:
Fair enough; I don't feel strongly enough to block merging on this, so it's fine to leave as-is.